client/editSchedule.js

```diff
@@ -66,22 +33,35 @@ function getScheduledRounds(competitionId) {
       title: 1,
     }
   }).fetch();
-  return rounds;
-}
-function getUnscheduledRounds(competitionId) {
-  var rounds = Rounds.find({
-    competitionId: competitionId,
-    startMinutes: {$exists: false}
-  }, {
+  // Round that are scheduled for times outside of the range shown by our calendar
+  // are considered unscheduled.
+  var competition = Competitions.findOne(competitionId, {
     fields: {
-      eventCode: 1,
-      roundCode: 1,
-
-      durationMinutes: 1,
-      nthDay: 1,
-      title: 1,
+      numberOfDays: 1,
+      calendarStartMinutes: 1,
+      calendarEndMinutes: 1,
     }
-  }).fetch();
+  });
+  function roundIsScheduled(round) {
+    if(typeof(round.nthDay) === "undefined" || typeof(round.startMinutes) === "undefined" || typeof(round.durationMinutes) === "undefined") {
```

Comment:
Hmmm... I think so. I'm in the habit of using the typeof(...) === "undefined", because I believe that's the only way to check if variables are undefined. These aren't variables, though, they're attributes of an object. I'll change it to your cleaner suggestion.